samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/RuntimeTestRunner.xaml

```diff
@@ -8,5 +8,5 @@
 	  mc:Ignorable="d"
 	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-	<rtt:UnitTestsControl />
+	<rtt:UnitTestsControl Margin="0,20,0,0" />
```

Comment:
How come we need a top margin here? Is it just to look nicer or is it not respecting the safe area?

Reply:
The safe area pushed the burger menu down so it's on top of the Run button
![image](https://github.com/unoplatform/uno.toolkit.ui/assets/7770915/27c606f8-cada-4968-9b32-03b51c164eb8)